### PR TITLE
chore: update version of `@umijs/preset-react` to 2.x in create-umi-app

### PR DESCRIPTION
**Pull request by awmleer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: packages/create-umi-app/templates/AppGenerator/package.json.tpl [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
-    "@umijs/preset-react": "1.x",
+    "@umijs/preset-react": "2.x",
```
Comment (xiaohuoni) [inline packages/create-umi-app/templates/AppGenerator/package.json.tpl:31]: 感觉应该用最新的？

Comment (awmleer) [reply packages/create-umi-app/templates/AppGenerator/package.json.tpl:31]: `lastest` 么？不能直接写的

Comment (xiaohuoni) [reply packages/create-umi-app/templates/AppGenerator/package.json.tpl:31]: 我的意思是大概这样子 `^2.x.x` 这个没必要锁包？

Comment (awmleer) [reply packages/create-umi-app/templates/AppGenerator/package.json.tpl:31]: 这两个写法是等价的啊

Comment (xiaohuoni) [reply packages/create-umi-app/templates/AppGenerator/package.json.tpl:31]: 哦

**Review by xiaohuoni — APPROVED** [review]

**Pull request closed without merging**